VmClassImpl.typeDefinition NPE when calling toString

NPE when calling `toString` on a Node:

```
Caused by: java.lang.NullPointerException: Cannot invoke "org.qbicc.type.definition.LoadedTypeDefinition.getInternalName()" because "this.typeDefinition" is null
	at org.qbicc.interpreter.impl.VmClassImpl.getName(VmClassImpl.java:255)
	at org.qbicc.interpreter.impl.VmClassImpl.toString(VmClassImpl.java:562)
	at java.base/java.lang.String.valueOf(String.java:4215)
	at java.base/java.lang.StringBuilder.append(StringBuilder.java:169)
	at org.qbicc.graph.literal.ObjectLiteral.toString(ObjectLiteral.java:62)
	at org.qbicc.graph.AbstractBinaryValue.toString(AbstractBinaryValue.java:46)
	at org.qbicc.graph.AbstractBooleanCompare.toString(AbstractBooleanCompare.java:9)
	at org.qbicc.graph.If.toString(If.java:106)
	at org.qbicc.graph.AbstractNode.toString(AbstractNode.java:42)
	at java.base/jdk.internal.reflect.GeneratedMethodAccessor18.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.jboss.byteman.rule.expression.MethodExpression.interpret(MethodExpression.java:366)
	... 28 more
```

--- a/interpreter/src/main/java/org/qbicc/interpreter/impl/VmPrimitiveClassImpl.java
+++ b/interpreter/src/main/java/org/qbicc/interpreter/impl/VmPrimitiveClassImpl.java
@@ -39,6 +39,11 @@ class VmPrimitiveClassImpl extends VmClassImpl implements VmPrimitiveClass {
     }
 
     @Override
+    public String getName() {
+        return simpleName;
+    }
+
+    @Override
     VmArrayClassImpl getArrayClass() {
         return (VmArrayClassImpl) arrayTypeDefinition.getVmClass();
     }

